spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -485,7 +486,7 @@ def add_electrodes(recording: se.RecordingExtractor, nwbfile=None, metadata: dic
         else:
             nwb_elec_ids = nwbfile.electrodes.id.data[:]
         for metadata_column in metadata['Ecephys']['Electrodes']:
-            if metadata_column['name'] not in nwbfile.electrodes.colnames:
+            if nwbfile.electrodes is None or metadata_column['name'] not in nwbfile.electrodes.colnames:
```

Comment:
This is the hotfix to add_electrodes() when running on a fresh or defaulted NWBFile.